pkg/handler/collector/file/file_test.go

```diff
@@ -92,6 +112,7 @@ func Test_fileCollector_RetrieveArtifacts(t *testing.T) {
 				lastChecked: tt.fields.lastChecked,
 				poll:        tt.fields.poll,
 				interval:    tt.fields.interval,
+				useBlobPath: tt.fields.useBlobPath,
```

Comment:
I looked more closely at this and decided that there was no change for me to make here.

1. Calling `NewFileCollector()` instead of instantiating the struct directly does not allow one to set `lastChecked`.
2. I think that if the error handler `eh()` does not get executed (and I think `eh()` usually or maybe always _does not_ get executed when the "with canceled poll" testcase is run) then the value of `wantErr` is not important. I am not sure how big a deal this is; I spent time trying to understand all of the execution and interaction here but after a while I didn't come to a conclusion and also decided it was unimportant. So, I will resolve this conversation.